wazo_call_logd/tests/test_generator_scenarios.py

```diff
@@ -772,3 +772,224 @@ def test_bad_apple(self, cels: list[CEL]):
+            HANGUP                    | 1738482679.53 | 1738482670.36 | 2025-02-02 11:21:21.670731+03:30 | John Conner                | 0012345678910                        | 447b36i8                             | wazo_wait_for_registration                                | Local/447b36i8@wazo_wait_for_registration-0000000c;2                  |                 |                                                                                                                                              | {"hangupcause":16,"hangupsource":"","dialstatus":""}
+            CHAN_END                  | 1738482679.53 | 1738482670.36 | 2025-02-02 11:21:21.670731+03:30 | John Conner                | 0012345678910                        | 447b36i8                             | wazo_wait_for_registration                                | Local/447b36i8@wazo_wait_for_registration-0000000c;2                  |                 |                                                                                                                                              |
+            CHAN_END                  | 1738482679.49 | 1738482670.36 | 2025-02-02 11:21:21.886307+03:30 | John Conner                | 0012345678910                        | s                                    | ctx-Company-internal-3344220c-6383-4581-9696-15055ec303e3 | PJSIP/uqe8wu8g-0000000b                                               | AppDial2        | (Outgoing Line)                                                                                                                              |
+            LINKEDID_END              | 1738482679.49 | 1738482670.36 | 2025-02-02 11:21:21.886307+03:30 | John Conner                | 0012345678910                        | s                                    | ctx-Company-internal-3344220c-6383-4581-9696-15055ec303e3 | PJSIP/uqe8wu8g-0000000b                                               | AppDial2        | (Outgoing Line)                                                                                                                              |
+        '''
+    )
+    def test_incoming_call_to_queue_no_answer(self, cels):
+        call_logs = self.generator.call_logs_from_cel(cels)
+        assert call_logs
+        assert len(call_logs) == 1
+
+        call_log, = call_logs
+        expected_properties = dict(
+            source_name='John Conner',
+            source_exten='0012345678910',
+            requested_exten='8083339999',
+            destination_exten='8083339999',
+            destination_name='SUPPORT',
+            direction='inbound',
+            destination_details={
+                'type': 'queue',
+                'queue_id': 1,
+                'queue_name': 'SUPPORT'
+            }
+        )
+        assert all(getattr(call_log, 'destination_details_dict' if expected_prop == 'destination_details' else prop) == expected_value for expected_prop, expected_value in expected_properties)
```

Comment:
c.f. https://zuul.wazo.community/zuul/t/local/build/9337dde9802347b5afc5f37e758de481/log/tox/py39/2-commands[0].log

```suggestion
        assert all(getattr(call_log, 'destination_details_dict' if expected_prop == 'destination_details' else prop) == expected_value for expected_prop, expected_value in expected_properties.items())
```